### PR TITLE
Update - 'Plasma' mentions removed

### PR DESCRIPTION
**Pull request by EmpieichO**
Update - 'Plasma' mentions removed in 4 docs.

### CODE REVIEW

File: docs/pos/concepts/tokens/pol.md [hunk 1]
```diff
@@ -68,9 +68,9 @@ Yes, the migration contract includes a feature known as "unmigration." This allo
 
 ## Bridging mechanisms
 
-### How does the modified Plasma Bridge function?
+### How does the modified Bridge function?
```
Comment (kmurphypolygon) [inline docs/pos/concepts/tokens/pol.md]:
```suggestion
### How does the modified bridge function?
```

File: docs/pos/concepts/tokens/pol.md [hunk 2]
```diff
@@ -68,9 +68,9 @@ Yes, the migration contract includes a feature known as "unmigration." This allo
...
 
-The Plasma Bridge will undergo modifications, with community approval, to change the native token of Polygon PoS to the new POL token. Specifically, the following changes are being proposed:
+The Bridge will undergo modifications, with community approval, to change the native token of Polygon PoS to the new POL token. Specifically, the following changes are being proposed:
```
Comment (kmurphypolygon) [inline docs/pos/concepts/tokens/pol.md]:
```suggestion
The bridge will undergo modifications, with community approval, to change the native token of Polygon PoS to the new POL token. Specifically, the following changes are being proposed:
```

File: docs/pos/overview.md [hunk 3]
```diff
@@ -9,28 +9,27 @@ Polygon PoS will continue to be foundational infrastructure for a wide array of
...
 
 1. **User initiates transaction**: On the Polygon PoS chain, typically via a smart contract function call.
-2. **Validation by public plasma checkpoint nodes**: These nodes validate the transaction against the Polygon chain's current state.
+2. **Validation by public checkpoint nodes**: These nodes validate the transaction against the Polygon chain's current state.
 3. **Checkpoint creation and submission**: A checkpoint of the validated transactions is created and submitted to the core contracts on the Ethereum mainnet.
 4. **Verification by core contracts**: Core contracts verify the checkpoint's validity, with the added security of fraud proofs. 
 5. **Transaction execution**: Upon successful verification, the transaction is executed and state changes are committed to the Polygon sidechain.
-6. **Asset transfer (optional)**: If needed, assets can be transferred back to the Ethereum mainnet via the Plasma Exit Queue in the core contracts.
+6. **Asset transfer (optional)**: If needed, assets can be transferred back to the Ethereum mainnet via the Exit Queue in the core contracts.
```
Comment (kmurphypolygon) [inline docs/pos/overview.md]:
```suggestion
6. **Asset transfer (optional)**: If needed, assets can be transferred back to the Ethereum mainnet via the exit queue in the core contracts.
```

File: docs/pos/overview.md [hunk 4]
```diff
@@ -9,28 +9,27 @@ Polygon PoS will continue to be foundational infrastructure for a wide array of
...
 
-The core contracts on the Ethereum mainnet incorporate two key features for security and functionality: fraud proofs and the Plasma Exit Queue. Fraud proofs act as a security layer, enabling the validation of transactions and state changes to ensure transparency and security across operations. The Plasma Exit Queue manages the safe and efficient transfer of assets back to the Ethereum mainnet, allowing users to seamlessly move assets between the Polygon PoS chain and Ethereum without compromising data integrity or security.
+The core contracts on the Ethereum mainnet incorporate two key features for security and functionality: fraud proofs and the Exit Queue. Fraud proofs act as a security layer, enabling the validation of transactions and state changes to ensure transparency and security across operations. The Exit Queue manages the safe and efficient transfer of assets back to the Ethereum mainnet, allowing users to seamlessly move assets between the Polygon PoS chain and Ethereum without compromising data integrity or security.
```
Comment (kmurphypolygon) [inline docs/pos/overview.md]:
```suggestion
The core contracts on the Ethereum mainnet incorporate two key features for security and functionality: fraud proofs and the exit queue. Fraud proofs act as a security layer, enabling the validation of transactions and state changes to ensure transparency and security across operations. The exit queue manages the safe and efficient transfer of assets back to the Ethereum mainnet, allowing users to seamlessly move assets between the Polygon PoS chain and Ethereum without compromising data integrity or security.
```

File: docs/tools/matic-js/advanced/exit-util.md [hunk 5]
```diff
@@ -47,9 +47,9 @@ const proof = await exitUtil.buildPayloadForExit(
 
 ```
 
-### Generating proof using bridge client
+### Generating proof using a bridge client
 
-Every bridge client including **POSClient**, **PlasmaClient** exposes `exitUtil` property.
+Every bridge client including **POSClient** exposes `exitUtil` property.
```
Comment (kmurphypolygon) [inline docs/tools/matic-js/advanced/exit-util.md]:
```suggestion
Every bridge client, including `POSClient` exposes the `exitUtil` property.
```

**Review by kmurphypolygon — CHANGES_REQUESTED** [review]
Looks good. I wonder about exit queue, seems to be a general thing rather than a proper noun. Also bridge.

**Review by EmpieichO — COMMENTED** [review]
changes accepted

**Review by kmurphypolygon — APPROVED** [review]

**Pull request merged**